Actions/IncrementVersionNumber/IncrementVersionNumber.psm1

```diff
@@ -0,0 +1,94 @@
+﻿<#
+    .Synopsis
+        Changes a setting value in a settings file.
+    .Parameter settingsFilePath
+        Path to the settings file.
+    .Parameter settingName
+        Name of the setting to change.
+    .Parameter newValue
+        New value of the setting.
+    .Parameter incremental
+        If set, the new value will be added to the old value. The old value must be a version number.
+#>
+function Set-SettingInFile($settingsFilePath, $settingName, $newValue, [switch] $incremental) {
+    if (-not (Test-Path $settingsFilePath)) {
+        throw "Settings file ($settingsFilePath) not found."
+    }
+
+    Write-Host "Reading settings from $settingsFilePath"
+    $settingFileContent = Get-Content $settingsFilePath -Encoding UTF8 | ConvertFrom-Json
+
+    if (-not ($settingFileContent.PSObject.Properties.Name -eq $settingName)) {
+        Write-Host "Setting $settingName not found in $settingsFilePath"
+        return
+    }
+
+    $oldValue = $settingFileContent.$settingName
+
+    if ($incremental) {
+        $oldValue = [System.Version]$oldValue
+        $newValue = [System.Version]$newValue
+
+        $newValue = "$($newValue.Major + $oldValue.Major).$($newValue.Minor + $oldValue.Minor)"
```

Comment:
This part doesn't make much sense to me. Why would anyone bump their version with  "+1.2", e.g. from 2.1 to 3.3?
Even "+1.0" seems unlikely. 

@freddydk what the reasoning behind this implementation?

Reply:
My suggestion here is to allow bumping the minor version, e.g. "+0.1" would change "1.2" to "1.3", or bumping to the next major version: "+1" would change "1.2" to "2.0".

What do you think? It's a design change though.

Reply:
You are probably right - +1.0 seems strange.
We can make this change to basically only support:
- major.minor
- +1 = increment major and set minor to 0
- +0.1 = increment minor
Meaning that we don't support

Reply:
Implemented.